phpstan.neon

```diff
@@ -14,10 +14,6 @@ parameters:
     message: "#^Left side of || is always false.$#"
     count: 6
     path: lib/Client.php
-  -
```

Comment:
I think we should drop the phpVersion from this config file. Since phpstan 2.x it will lookup the composer.json

Reply:
> I think we should drop the phpVersion from this config file. Since phpstan 2.x it will lookup the composer.json

That made it fail on PHP 8.4 https://github.com/sabre-io/http/actions/runs/12631715455/job/35193980626?pr=255

I suppose that I should have a look at the reported things?

Reply:
I would ignore `CurlHandle, resource given` and such for now

Reply:
by php-version errors are usually separated into separate baselines

https://github.com/phpstan/phpstan-src/blob/2f712479fe1aa86f618a49fe4f36a3531230484b/build/phpstan.neon#L10

if this sounds like too much work, we can revert to adding the php version into phpstan.neon and wait until I am able to improve the phpstan multi phpversion story with https://staabm.github.io/2024/11/28/phpstan-php-version-in-scope.html